authN-authZ/auth-istio/README.md

```diff
@@ -330,3 +398,15 @@ sudo sed -i '1i\127.0.0.1       chatqna-ui.com' /etc/hosts
 Open browser with address `"chatqna-ui.com:${INGRESS_PORT}"` if using GMC based deployment. Otherwise, open the browser with address `"chatqna-service.com:${INGRESS_PORT}"`.
```

Comment:
we never told folks how to find the address of chatqna-ui.com.  the way it is written it feels like a fully qualified domain name like google.com!

Reply:
I didn't validate UI , this part may need @Ruoyu-y  to check and update in another PR.

Reply:
'chatqna-ui.com' is indeed a domain name that MUST be used to connect to the UI service, as it has been set in the istio configuration. And there's also one step to set the ip address within the /etc/hosts file to help resolve the domain name. So it should be fine to access the UI service from the domain name.

```
Add both host names for ChatQnA UI and backend service into /etc/hosts

sudo sed -i '1i\127.0.0.1       chatqna-service.com' /etc/hosts
sudo sed -i '1i\127.0.0.1       chatqna-ui.com' /etc/hosts
```

Reply:
@XinyaoWa would you please add the /etc/hosts change that @Ruoyu-y mentions above.
"Edit the /etc/hosts file on the computer from where you are working/browsing to bypass the need for a DNS server to resolve your FQDN (fully qualified domain name). 
vi /etc/hosts
<ip-address-of-gateway>     chatqna-service.com
<ip-address-of-gateway>     chatqna-ui.com

By setting you routing for FQDN based services you can restrict access to only services you want to expose in your Kubernetes cluster.
"

Reply:
Code already exist in readme, I have add the explanation you provided.